.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - 0.8
```

Comment:
I think this'll start going once this file hits `master`. https://travis-ci.org/duckduckgo/duckduckgo-privacy-extension